### DISCUSSION TITLE
Is there something like "css grid" currently in the app?

### DISCUSSION THREAD

**Discussion by victoruxui** [uxbox/uxbox#2916] [Ideas]
I'm looking to get closer to the real css code with something like css grid but not sure if that's currently in the ap or even if it's in the roadmap?

**Comment by elhombretecla** [comment]
Hi victor!!! Thank you for your feedback

Could you move this conversation to our [community forum](https://community.penpot.app/)? Is a more open space to the rest of the users. 

Thanks!
